### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by bnaluz**

### CODE REVIEW

File: backend/db/migrations/20240312000713-create-user.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        updatedAt: {
+          allowNull: false,
+          type: Sequelize.DATE,
+          defaultValue: Sequelize.literal('CURRENT_TIMESTAMP'),
+        },
+      },
+      options
+    );
+  },
+  async down(queryInterface, Sequelize) {
```
Comment (AlanDeleon88) [inline backend/db/migrations/20240312000713-create-user.js:55]: You need to make sure you are including your options object in your drop statement as well. Take a look at the code shown in phase 3.

**Review by AlanDeleon88 — CHANGES_REQUESTED** [review]
Take a look at my comment on your user table migration. Fix that and make another PR. If that checks out, you can merge.

File: backend/db/migrations/20240312000713-create-user.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    await queryInterface.dropTable('Users');
```
Comment (AlanDeleon88) [inline backend/db/migrations/20240312000713-create-user.js]: You need to make sure you are including your options object in your drop statement as well. Take a look at the code shown in phase 3.

image

That options object lets your render application know which schema to delete from. All of your projects deployed on render share the same database, so when you are migrating you need to let render know which schema you are migrating to.

Comment (bnaluz) [reply backend/db/migrations/20240312000713-create-user.js]: i updated the migration file, is it reflecting on your end?

**Review by AlanDeleon88 — APPROVED** [review]

File: backend/db/migrations/20240312000713-create-user.js (hunk 2 above)
Comment (AlanDeleon88) [reply backend/db/migrations/20240312000713-create-user.js]: looks good. feel free to merge.

**Pull request merged**